test/AstAnalyser.spec.js

```diff
@@ -225,6 +225,18 @@ describe("AstAnalyser", (t) => {
       assert.strictEqual(preparedSource, "\nconst yo = 'foo'\n");
     });
   });
+
+  describe("constructor", () => {
+    it("should not throw an error when instantiated without a custom parser", () => {
+      assert.doesNotThrow(() => {
+        const analyser = getAnalyser();
```

Comment:
Actually this test is wrong.
The `getAmalyser()` function returns the `AstAnalyser` instanciated with `JsSourceParser()`.

```js
analyser = new AstAnalyser(new JsSourceParser());
```

```suggestion
        const analyser = new AstAnalyser();
```